src/SFA.DAS.ContentApi.Database/Tables/Content.sql

```diff
@@ -0,0 +1,10 @@
+﻿CREATE TABLE [dbo].[Content]
+(
+	[Id] INT NOT NULL PRIMARY KEY, 
+    [ContentTypeId] INT NOT NULL, 
+    [Content] TEXT NULL, 
```

Comment:
Using the TEXT data type is not recommend by MS the VARCHAR(MAX) / NVARCHAR(MAX) is recommended.